AK/RecursionDecision.h

```diff
@@ -1,23 +0,0 @@
-/*
- * Copyright (c) 2021, sin-ack <sin-ack@protonmail.com>
- *
- * SPDX-License-Identifier: BSD-2-Clause
- */
-
-#pragma once
-
-#include <AK/Platform.h>
-
-namespace AK {
-
-enum class RecursionDecision {
-    Recurse,
-    Continue,
-    Break,
-};
```

Comment:
I think this was effectively evaded by use of `Web::TraversalDecision` in any place in the code where we deal with tree traversal. We might at some point want to move it to `AK` again since that seems to be the best place to deal with trees. (Just a remark for reference, no need to change anything in this PR)